api/v2.0/swagger.yaml

```diff
@@ -8368,7 +8424,12 @@ definitions:
         default: ""
         description: Indicate the healthy of the registration
         example: "healthy"
-
+      capabilities:
+        type: object
+        description: Indicate the scanner's properties such as support_vulnerability or support_sbom
```

Comment:
Indicates the capabilities of the scanner, e.g. support_vulnerability or support_sbom.